packages/components/psammead-media-indicator/README.md

```diff
@@ -20,6 +20,15 @@ The `MediaIndicator` component provides a 'play' or 'audio' icon as well as a du
 | topStory      | boolean | No      | false   | true                         |
 | service | string | Yes | N/A | `'news'` |
 
+### Supported `type`s
+
+<!-- prettier-ignore -->
```

Comment:
This isnt working in the markdown view
![image](https://user-images.githubusercontent.com/11341355/61324382-2f681380-a80a-11e9-83e7-439ed5d7d020.png)

Reply:
seems that svgs are an xss vector so github blocks them in READMEs 🙃 https://stackoverflow.com/a/25606546

will just revert to showing a list of types, not their previews